### PR TITLE
WEBAPP-640: Pull malte locales

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

**Comment by maxammann** [comment]
https://github.com/Integreat/integreat-webapp/pull/397/files/8477abd382e542f83d32d5b4cd86473b6aed5129..b49da950773b633d155b4dd58212cd8fcb0233e7

File: src/modules/i18n/loadLocales.js [hunk 1]
```diff
@@ -27,13 +27,10 @@ const transformLocales = (locales: LocalesType): TransformedLocalesType => reduc
 )
 
 const loadLocales = (): TransformedLocalesType => {
-  // TODO WEBAPP-640: Uncomment this section
-  // const localesOverride = buildConfig().localesOverride
+  const localesOverride = buildConfig().localesOverride
   // // If keys are missing in 'defaultLocales', merge does not include those
   // // https://lodash.com/docs/4.17.15#merge
```
Comment (Schedulaar) [inline src/modules/i18n/loadLocales.js:32]: remove duplicate comment syntax

**Review by Schedulaar — APPROVED** [review]
didn't check the translations...

File: src/modules/i18n/loadLocales.js (hunk 1 above)
Comment (maxammann) [reply src/modules/i18n/loadLocales.js:32]:
```suggestion
  // If keys are missing in 'defaultLocales', merge does not include those
  // https://lodash.com/docs/4.17.15#merge
```

**Comment by maxammann** [comment]
Locales are already reviewed

File: locales/tools/manage [hunk 2]
```diff
@@ -138,14 +138,31 @@ const writeJsonFromCsv = (locales, toPath, sourceLanguage) => {
...
 
+    const byLanguageModules = fromPairs(csvs.map(csvFile => [path.basename(csvFile, '.csv'), loadModules(csvFile, 'target_language')]))
+
     const sourceModules = loadModules(csvs[0], 'source_language')
 
+    // Show which source languages differ
+    csvs.forEach((csv, index) => {
+      if (!isEqual(loadModules(csv, 'source_language'), sourceModules)) {
+        console.log('source language 1', sourceModules)
+        console.log()
```
Comment (Taggotty) [inline locales/tools/manage:154]: why the space?

**Review by Taggotty — APPROVED** [review]

File: locales/tools/manage (hunk 2 above)
Comment (maxammann) [reply locales/tools/manage:154]: That was done by @klinzo and is not part of this PR

**Pull request merged**